js_modules/dagit/src/execute/ExecutionStartButton.tsx

```diff
@@ -89,32 +90,43 @@ export default class ExecutionStartButton extends React.Component<
   }
 }
 
-const Wrapper = styled.div<{ disabled?: boolean }>`
+const Wrapper = styled.div<{ state: "ready" | "disabled" | "running" }>`
```

Comment:
Mmm we could potentially re-use the pipeline state enum here, but the execute button state is a bit more transient since it becomes "ready" again as soon as the pipeline is kicked off, and "disabled" means the dagit process is gone. I'll convert it into an enum in this file though.